nix/scripts/create-container-dump.sh

```diff
@@ -36,5 +36,7 @@ while IFS= read -r image; do
           docker://$image_trimmed docker-archive:${image_path} --additional-tag $image
       fi
       echo "${image_filename}.tar" >> $(realpath "$1")/index.txt
+      # passing image and $1 = output dir
```

Comment:
you should assign $1 to a variable name that is more descriptive, so you don't have to add this comment.